dask_expr/tests/test_collection.py

```diff
@@ -15,30 +17,34 @@
 from dask_expr._reductions import Len
 from dask_expr.datasets import timeseries
 
+# Import backend DataFrame library to test
+BACKEND = dask.config.get("dataframe.backend", "pandas")
+lib = importlib.import_module(BACKEND)
+
 
 @pytest.fixture
-def pdf():
-    pdf = pd.DataFrame({"x": range(100)})
-    pdf["y"] = pdf.x * 10.0
-    yield pdf
+def bdf():
+    bdf = lib.DataFrame({"x": range(100)})
+    bdf["y"] = bdf.x * 10.0
+    yield bdf
```

Comment:
Note that I renamed `pdf` to `bdf` for "backend" DataFrame, but also happy to continue using the `pdf` convention for both backends.

Reply:
I have a preference to stick with `pdf` and a general default for pandas naming.

Reply:
Sounds good to me. Thanks